requirements.txt

```diff
@@ -72,3 +72,4 @@ flask-marshmallow==0.8.0
 ipython<6,>=5.4.1
 prompt-toolkit<2.0.0,>=1.0.14
 packaging>=16.8,<17
+flask-webpackext==0.1.0
```

Comment:
What about also installing pywebpack from your fork here for now until your change is in a released version of it?